evap/staff/templates/staff_semester_view.html

```diff
@@ -287,15 +287,15 @@ <h3>
                         <div>
                             <button name="target_state" value="prepared" type="submit" class="btn btn-sm btn-light"
                                 data-toggle="tooltip" data-placement="top" title="{% trans 'Evaluations in preparation or approved by editors' %}">
-                                <span class="{{ 'new'|approval_state_icon }}"></span>
-                                <span class="{{ 'editor_approved'|approval_state_icon }}"></span>
+                                <span class="{{ 'new'|state_from_english|approval_state_icon }}"></span>
```

Comment:
I guess `approval_state_icon` could work with the string constants, but I thought that it would be more consistent if everything was based on `Evaluation.State`. Then, one would convert from a string to an int as early as possible to use all of the "library" with the int.

I could perform the conversion in the icon function too though if you think that's a better fit

Reply:
Nono, not the string constants, the integer constants! :) Maybe you didn't do that because you don't have an `evaluation` object available? Is there any way to make the enum available in the templates regardless?

Reply:
Oh, now I get it :laughing: yeah, I just passed `Evaluation` now :+1: